lua/astrocommunity/test/nvim-coverage/init.lua

```diff
@@ -1,6 +1,44 @@
+---@type LazySpec
 return {
   "andythigpen/nvim-coverage",
   event = "User AstroFile",
-  dependencies = { "nvim-lua/plenary.nvim" },
-  opts = {},
+  opts = {
+    auto_reload = true,
+  },
+  dependencies = {
+    "nvim-lua/plenary.nvim",
+    {
+      "AstroNvim/astroui",
+      opts = {
+        icons = {
+          Tests = "󰗇",
+          Coverage = "",
+        },
+      },
+    },
+    {
+      "AstroNvim/astrocore",
+      optional = true,
+      opts = function(_, opts)
+        local coverage = require "coverage"
```

Comment:
I'll need to double check but having this `require "coverage"` out here instead of in the functions e.g

`maps.n[coverage_prefix .. "t"] = { function() require("coverage").toggle() end, desc = "Toggle coverage" }`

Will force this to always load instead of lazy loading it, since astrocore is always and will always require it right away.

Reply:
Ok, tell me if we need to change it, I understand the issue.

Reply:
Just tried it out it's preventing lazy loading and always loading it in, can you inline the require coverage? thanks :)